frontend/packages/operator-lifecycle-manager/src/components/operand/utils.spec.ts

```diff
@@ -14,22 +14,14 @@ import { HIDDEN_UI_SCHEMA } from './const';
 describe('getJSONSchemaOrder', () => {
```

Comment:
Formatting and added another unit test to check an edge case in the `hasNoFields` helper.